### PR TITLE
[WIP] Free energy gradients

### PR DESCRIPTION
**Pull request by maxentile**
## Description
attempting to get gradients of free energy estimates w.r.t. ANI neural net parameters by differentiating through pymbar computePerturbedFreeEnergies

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [x] housekeeping stuff (unused imports, add some todos, remove xyz-file-writing functions, update some docstrings)
  - [x] compute a free energy difference with pymbar + torchani
  - [x] write a pytorch function to compute perturbed free energies using equilibrium snapshots
  - [ ] compute gradient of a free energy difference

## Status

Debugging where gradient information gets lost inside of the [LinearAlchemicalSingleTopologyANI](https://github.com/choderalab/neutromeratio/blob/4539b997980522ab63d03494eb467e04e0811657/neutromeratio/ani.py#L319).calculate_energy function -- current suspicion: maybe pytorch doesn't trace through the simtk.unit conversions...

### CODE REVIEW

**Comment by maxentile** [comment]
>Are you using units just for convenience? It might be useful just to adopt fully self-consistent dimensionless units for this particular code

Just for convenience -- if this is the culprit, that'll be the way to go! Checking whether removing units resolves this issue...
